src/app.ts

```diff
@@ -264,17 +268,22 @@ if (ENV.DEBUG_THEME) {
         } catch (e: any) {
           console.log(`Failed get theme`, e.message)
         }
+      }
+      setInterval(() => {
+        intervalCallback().catch((err: unknown) => {
+          console.error(err)
+        })
```

Comment:
Optional: This should really be `makePeriodicTask`, since that avoids overlapping work in cases where the `async` function takes more that 3s.